contracts/v0.8.9/Issue552_Reproduction.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity ^0.8.9;
+pragma experimental ABIEncoderV2;
+
+
+library Issue552_Observer {
```

Comment:
Can we simplify this example? I guess something as simple as `uint256[65535] ints` triggers it already.

Reply:
I would propose something simple as:

```
pragma solidity ^0.8.9;
pragma experimental ABIEncoderV2;

contract Issue552_Reproduction {
  uint256[65535] public ints;

  function input(uint256[10] calldata values) public pure {}
}
```

Then we need a runtime test for `input` to ensure @zemse's comment is fixed.

Reply:
```
uint256[65535] public ints;
```

☝ this doesn't emit the type, only the getter function if I see things correctly. I removed some redundant code from the reproduction though.